### ISSUE TITLE
Migrating external accounts from greenlight to greenlight-v3

### ISSUE THREAD

**Issue by dashohoxha** [bigbluebutton/greenlight#5147]
I followed the instructions at https://docs.bigbluebutton.org/greenlight/v3/migration/ and everything worked well. However most of the user accounts are based on Google authentication and cannot login anymore on greenlight-v3. There are no instructions on the docs what to do with these accounts: https://docs.bigbluebutton.org/greenlight/v3/migration/#after-the-migration

Is there any way to convert these external accounts to normal accounts, so that they can reset the password and be able to login?

**Comment by ffdixon** [comment]
Got your message.  The devs are working on another item at the moment; we'll update this issue when we've got more info.

**Comment by drudgede** [comment]
@dashohoxha I had the same problem because the rake migration script migrated by default users created by greenlight directly only. I found a fix as proposed in #5281. However, I have no Google users but only LDAP-provided ones but I guess, you can adapt the approach by checking the stored provider value in the greenlight v2 database and use this value as a parameter in the migration script.

**Comment by dashohoxha** [comment]
What I wanted to do was to migrate external (Google) accounts to normal accounts, so that they can reset the password and login with username and password. For a migrated external account, it was not possible to login, and it was not possible to reset the password.

Inspired by your fix, I looked at the postgres database and noticed that external google accounts have some value on the field `external_id`, and the other accounts have NULL instead. So, I tried to make this field NULL, and it worked; afterwards I was able to reset the password from the greenlight interface.

Here is a transcript of the steps:

```
root@bbb:~/greenlight-v3# docker-compose ps
root@bbb:~/greenlight-v3# docker-compose exec postgres bash
8a56f1987744:/# su - postgres
8a56f1987744:~$ psql
greenlight-v3-production=# \l
greenlight-v3-production=# \c greenlight-v3-production
greenlight-v3-production=# \dt
greenlight-v3-production=# \d users
greenlight-v3-production=# select name, email, password_digest, external_id, provider from users;
greenlight-v3-production=# update users set external_id=NULL where email='user@gmail.com';
greenlight-v3-production=# \q
8a56f1987744:~$ exit
8a56f1987744:/# exit
```

Maybe this can be documented somehow, or maybe migration scripts can be improved. I don't think that a normal user can do this comfortably. I was lucky to find this solution, and I had to google every step because I am not familiar with postgres.
